Services/BrokerService/Constants.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace BrokerService
+{
+    public static class Constants
```

Comment:
Do we really need another static class for variables that used In only one another class? 
Why don't we set word const in definition ? Like internal static int Depth = 5;
Name doesn't fit name convention

Reply:
> Do we really need another static class for variables that used In only one another class?
> Why don't we set word const in definition ? Like internal static int Depth = 5;
> Name doesn't fit name convention

Fix

Reply:
It's better to write fixed, I think :) 
Or I would think that you suggest me to fix it

Reply:
> It's better to write fixed, I think :)
> Or I would think that you suggest me to fix it


Corrected